deployments/daemon-set/nginx-ingress-with-prometheus.yaml

```diff
@@ -38,7 +38,7 @@ spec:
         args:
           - -nginx-configmaps=$(POD_NAMESPACE)/nginx-config
           - -default-server-tls-secret=$(POD_NAMESPACE)/default-server-secret
-         #- -v=3 # Enables extensive logging. Useful for trooublshooting.
+         #- -v=3 # Enables extensive logging. Useful for troublshooting.
```

Comment:
Just a little thing, I think you missed an `e` in all of the changes. Isn't it `troubleshooting`?